src/components/menu/MenuConnected.tsx

```diff
@@ -0,0 +1,144 @@
+import * as classNames from 'classnames';
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import * as _ from 'underscore';
+import {IReactVaporState, IReduxActionsPayload} from '../../ReactVapor';
+import {IReduxAction, ReduxConnect} from '../../utils/ReduxUtils';
+import {IItemBoxProps} from '../itemBox/ItemBox';
+import {ListBox} from '../listBox/ListBox';
+import {Svg} from '../svg/Svg';
+import {addMenu, removeMenu, toggleMenu} from './MenuActions';
+import {IMenuState} from './MenuReducers';
+
+export interface IMenuOwnProps {
+    items: IItemBoxProps[];
+    classes?: string[];
+    id: string;
+    svgName?: string;

```

Comment:
Instead of having a svg, maybe we should use the children and use the tag like so:

```jsx
<MenuConnected><Svg svgName='svg-name' svgClass='fill-medium-blue icon mod-lg' /></MenuConnected>
<MenuConnected>Some String</MenuConnected>
<MenuConnected><OtherTag /></MenuConnected>
```